src/BlazorWebView/src/Maui/Android/AndroidMauiAssetFileProvider.cs

```diff
@@ -29,8 +29,8 @@ public IDirectoryContents GetDirectoryContents(string subpath)
 		public IFileInfo GetFileInfo(string subpath)
 			=> new AndroidMauiAssetFileInfo(_assets, Path.Combine(_contentRootDir, subpath));
 
-		public IChangeToken? Watch(string filter)
-			=> null;
+		public IChangeToken Watch(string filter)
+			=> NullChangeToken.Singleton;
```

Comment:
cc/ @Eilon in case he has any objections here.

Reply:
I assume this is mechanically identical? What is the downside? Does it affect people doing explicit null checks?

Reply:
Oh I see it's a non-null change token that is morally equivalent to a null change token. I think this is fine. It doesn't break or change any contract or functional behavior. Plus, in practice no one uses these tokens obviously because they were already null beforehand.